### PR TITLE
Initial PR for submission

### PR DESCRIPTION
**Pull request by kasiagawlak**
@richardtape Hi Rich! Submitting this to establish a baseline, it will end up active on the VPFO sandbox once approved. I look forward to your feedback.

### CODE REVIEW

File: includes/hooks.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+// set up the default VPFO template as the starting page template for new pages
+function vpfo_set_default_template_on_new_page( $post_id, $post, $update ) {
+	// Only proceed if it's a page, it's not an update (new page), and it's in the admin area
+	if ( 'page' === $post->post_type && ! $update && is_admin() ) {
+		$default_template = 'vpfo-page.php'; // Replace with your custom template slug
+
+		// Check if the page has a template assigned; if not, set the default template
+		if ( get_post_meta( $post_id, '_wp_page_template', true ) === '' ) {
+			update_post_meta( $post_id, '_wp_page_template', $default_template );
```
Comment (richardtape) [inline includes/hooks.php]: Please can we have some sanitization here before it hits the database. absint() for the post ID and perhaps esc_html() for the value.

File: includes/hooks.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	}
+
+	// Check user permissions
+	if ( ! current_user_can( 'edit_post', $post_id ) ) {
+		return;
+	}
+
+	// Save the 'display alert' checkbox value
+	$display_alert = isset( $_POST['vpfo_display_alert'] ) ? '1' : '0';
+	update_post_meta( $post_id, '_vpfo_display_alert', $display_alert );
```
Comment (richardtape) [inline includes/hooks.php]: Please can we sanitize these values.

**Review by richardtape — CHANGES_REQUESTED** [review]
I really appreciate you folks meeting our coding styles and standards. There's just a couple of times where I think a little extra sanitization is needed. Once they're handled, I'm happy to get this onto our platform for you.

File: includes/hooks.php (hunk 1 above)
Comment (kasiagawlak) [reply includes/hooks.php]: @richardtape Sanitization added, please review.

File: includes/hooks.php (hunk 2 above)
Comment (kasiagawlak) [reply includes/hooks.php]: @richardtape Sanitization added, please review.

**Review by richardtape — APPROVED** [review]

**Pull request merged**